Development/nmos-cpp-registry/config.json

```diff
@@ -51,9 +51,16 @@
     // listen_backlog [registry, node]: the maximum length of the queue of pending connections, or zero for the implementation default (the implementation may not honour this value)
     //"listen_backlog": 0,
 
+    // registration_heartbeat_interval [registry, node]:
+    // [registry]: used in System API resource is04 object's heartbeat_interval field
+    // "Constants related to the AMWA IS-04 Discovery and Registration Specification are contained in the is04 object.
+    // heartbeat_interval defines how often Nodes should perform a heartbeat to maintain their resources in the Registration API."
+    // See https://specs.amwa.tv/is-09/releases/v1.0.0/docs/4.2._Behaviour_-_Global_Configuration_Parameters.html#amwa-is-04-nmos-discovery-and-registration-parameters
+    //"registration_heartbeat_interval": 5,
+
     // registration_expiry_interval [registry]:
     // "Registration APIs should use a garbage collection interval of 12 seconds by default (triggered just after two failed heartbeats at the default 5 second interval)."
```

Comment:
The spec is using the `should` rather than the `SHOULD`, maybe the spec ought to be changed?

Reply:
We must be misunderstanding each other. The spec v1.3.1, which this change makes reference to, says SHOULD as far as I can see.

Reply:
Do you mean we ought to be using https://specs.amwa.tv/is-04/releases/v1.3.1/docs/4.1._Behaviour_-_Registration.html#heartbeating instead of https://specs.amwa.tv/is-09/releases/v1.0.0/docs/4.2._Behaviour_-_Global_Configuration_Parameters.html#amwa-is-04-nmos-discovery-and-registration-parameters?

As https://specs.amwa.tv/is-09/releases/v1.0.0/docs/4.2._Behaviour_-_Global_Configuration_Parameters.html#amwa-is-04-nmos-discovery-and-registration-parameters is using `should` instead of `SHOULD`

Reply:
The first link to is-04/v1.3.1 **IS** the one you've changed it to? I'm so confused. 😂

Reply:
The comment on this setting was used to be related to Node implementation.  But once we have started using it for the Registry's System API, the comment has not been updated. All I have done was updating the Registry part to point to the System API reference.

Reply:
The 'master' comments have been the ones in **nmos/settings.h**, the example **config.json** files have been created from this, with the irrelevant properties removed but the comments unchanged.

I think it could make sense to add the link to IS-09 in **nmos/settings.h** as well as the IS-04 link if you'd like. At the moment you have updated the reference here in **nmos-cpp-registry/config.json** to the latest IS-04 version (not IS-09) but not updated the quote to match...